example/ios/Podfile

```diff
@@ -18,10 +18,10 @@ target 'example' do
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable these next few lines.
-  use_flipper!
-  post_install do |installer|
-    flipper_post_install(installer)
-  end
+  # use_flipper!
+  # post_install do |installer|
+  #   flipper_post_install(installer)
+  # end
```

Comment:
Is this commented out due to xcode 12 issues?  If so, let's undo for the PR once we have positive testing.

Reply:
indeed. i can revert this at the end.